cumulusci/salesforce_api/tests/test_metadata.py

```diff
@@ -405,34 +404,10 @@ def test_get_response_status_error_500(self):
 
         response = '<?xml version="1.0" encoding="UTF-8"?><id>1234567890</id>'
         self._mock_call_mdapi(api, response)
-        response_status = '<?xml version="1.0" encoding="UTF-8"?><foo>status</foo>'
         self._mock_call_mdapi(api, response, status_code)
 
         with self.assertRaises(MetadataApiError):
-            resp = api._get_response()
-
-    @responses.activate
-    def test_get_response_status_error_500(self):
```

Comment:
So this was a duplicate method name? We should keep the 2nd one, which would be the one that was actually getting executed before.

Reply:
The two methods were byte-for-byte identical - looks like an errant copy and paste.